README.md

```diff
@@ -901,6 +901,7 @@ cause a re-render of the component and will not be reflected on the table.
 | ------------- | --------- | --------- | ---------- | ---------------------------------------------------- |
 | `columns`    | `object[]` | - | ✓ | An array of column objects, as described above |
 | `data`    | `T[]` | - | ✓ | An array of objects that will be displayed in the table |
+| `checkedItems`    | `T[]` | - | ✓ | An array of objects that controls which rows are selected |
```

Comment:
Should this be required without a default?

Reply:
@StefKors the docs are wrong, it should be optional.